src/main/resources/application-prod.properties

```
@@ -0,0 +1,12 @@
+account.name = produser
+account.password = prod
+
+spring.datasource.username=root
+spring.datasource.password=masterpw
+spring.datasource.url=jdbc:mysql://192.168.47.150:3306/subway?serverTimezone=UTC&characterEncoding=UTF-8
+
+logging.level.root=ERROR
```

```suggestion
logging.level.nextstep.subway=info
```

보통 root보다는 어플리케이션의 레벨을 설정합니다.
또한 info 정도까지는 상용에 남기는 경우가 많습니다.

현재 파일 변경 내역에서 보면 많은 로그가 info로 찍혀있는데요,
사실 대부분은 debug 레벨에 가깝다고 보여집니다.

info는 어플리케이션의 설정이 바뀌어서 정보성으로 로그를 남겨야 할 때 사용하고는 합니다.
그래서 어플리케이션의 상태 변화를 파악하기 위해 info까지는 상용에 남겨두는게 보통이지요.